docs/docs/graphql-api.md

```diff
@@ -0,0 +1,242 @@
+---
+title: GraphQL API
```

Comment:
```suggestion
title: GraphQL API
tableOfContentsDepth: 2
```
The `params` and `returns` are pretty repetitive, so I'd recommend suppressing those. 2 seems to be the magic number when I run this locally.